util/listForSQL.js

```diff
@@ -0,0 +1,12 @@
+// given an array and 
+module.exports = (array, db) => {
```

Comment:
Except for wrapping each element in a call to `db.escape`, you've just reimplemented the `join` method on arrays, only probably not as efficiently as the runtime's implementation. I suggest you replace this whole procedure with `array.map(db.escape).join(',')`.